Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingList/OfferingsList.swift

```diff
@@ -100,6 +100,13 @@ struct OfferingsList: View {
                 }
                 .id(viewModel.presentedPaywall?.hashValue) //FIXME: This should not be required, issue is in Paywallview
         }
+        .fullScreenCover(item: $viewModel.presentedPaywall) { paywall in
+            PaywallPresenter(offering: paywall.offering, mode: paywall.mode, introEligility: introEligility)
+                .onRestoreCompleted { _ in
+                    viewModel.dismissPaywall()
+                }
+                .id(viewModel.presentedPaywall?.hashValue) //FIXME: This should not be required, issue is in Paywallview
```

Comment:
I hit a similar issue when working on the paywall reload, the reason being:

https://github.com/RevenueCat/purchases-ios/blob/main/RevenueCatUI/Templates/V2/PaywallsV2View.swift#L125-L137